### PR TITLE
[IN-23][Ember-OSF] Add computed field to user model

### PR DESCRIPTION
**Pull request by adlius**
## Purpose

This is the first of two PRs (the second is [here](https://github.com/CenterForOpenScience/ember-osf-preprints/pull/521)) to make sure preprint detail page will fall back to using `fullName` of contributors.

## Summary of Changes

Add a computed field `name` to `user` model. Add some tests.

## Side Effects / Testing Notes

None. Testing notes will be included in [the second PR](https://github.com/CenterForOpenScience/ember-osf-preprints/pull/521).

## Ticket

https://openscience.atlassian.net/browse/IN-23

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: addon/models/user.js [hunk 1]
```diff
@@ -39,5 +39,15 @@ export default OsfModel.extend({
 
     // Calculated fields
     profileURL: Ember.computed.alias('links.html'),
-    profileImage: Ember.computed.alias('links.profile_image')
+    profileImage: Ember.computed.alias('links.profile_image'),
+    name: Ember.computed('fullname', 'giveName', 'familyName', function () {
+        let fullName = this.get('fullName');
+        let givenName = this.get('givenName');
+        let familyName = this.get('familyName');
+        if (givenName && familyName){
```
Comment (alexschiller) [inline addon/models/user.js]: Minor: space after {
Someday eslint will catch that :)

**Comment by laurenbarker** [comment]
@adlius @alexschiller same thing for this PR. Can be based on/merged into `release/next-interfaces`

**Comment by adlius** [comment]
Changed base. Conflicts for CHANGELOG are resolved.

**Comment by alexschiller** [comment]
More changelog conflicts. otherwise should be g2g

**Pull request merged**